pkg/services/searchV2/bluge.go

```diff
@@ -196,6 +203,25 @@ func getDashboardPanelDocs(dash dashboard, location string) []*bluge.Document {
 	return docs
 }
 
+var ngramIndexAnalyzer = &analysis.Analyzer{
+	Tokenizer: tokenizer.NewWhitespaceTokenizer(),
+	TokenFilters: []analysis.TokenFilter{
+		token.NewLowerCaseFilter(),
+		token.NewEdgeNgramFilter(token.FRONT, 1, 7),
```

Comment:
we may want to tweak 7 up or down.  this prefix is mostly useful for typeahead, but the variations explode if you get too many